### DISCUSSION TITLE
does the path at the prompt automatically get shortened when the path is too long, even when `path_shorten` for `files` is set to false?

### DISCUSSION THREAD

**Discussion by znxuz** [ibhagwan/fzf-lua#1128] [Q&A] [answered]
Is this hardcoded somewhere in the code? Or is it configurable?

**Comment by znxuz** [comment]
Nvm, found the option `cwd_prompt` just 5 seconds after I posted this thread.

**Reply by ibhagwan** [reply, marked as answer]
You have even more control, I’m in the middle of adding auto documentation to all options via nvim-cmp, take a look at `cwd_prompt_shorten_len` and `cwd_prompt_shorten_val` for further control at:

https://github.com/ibhagwan/fzf-lua/blob/cmp_src/OPTIONS.md
